pci/src/configuration.rs

```diff
@@ -315,6 +316,7 @@ pub struct PciConfiguration {
 #[derive(Copy, Clone, PartialEq, Serialize, Deserialize)]
 pub enum PciBarRegionType {
     Memory32BitRegion = 0,
+    #[cfg(target_arch = "x86_64")]
```

Comment:
This adds so little code I would prefer if we could keep the I/O region non-conditional.